app/src/utils/errorUtils.ts

```diff
@@ -9,13 +9,13 @@ const isErrorWithMessage = (error: unknown): error is { message: string } => {
 /**
  * Regex to match an error message in a Relay mutation error.
  * See example below where "Actual Error Message" is the error message.
- * Matches the string after the first occurrence of "message": " and before the next occurrence of " with double or single quotes.
+ * Matches the string after the first occurrence of "message": " and before the next occurrence of " with double quotes.
  * @example
  * ```
  * "Error fetching GraphQL query 'MutationName' with variables '{'input':{'var1': 'test'}': [{'message':'Actual Error Message','locations':[{'line':4,'column':3}],'path':['responsePath']}]"
  * ```*
  */
-const mutationErrorRegex = /['"']message['"']:\s*['"']([^'"']+)['"']/g;
+const mutationErrorRegex = /["]message["]:\s*["]([^""]+)["]/g;
```

Comment:
@Parker-Stafford was there a particular use case you had in mind for single quotes? It prevents us from capturing messages like `A prompt named 'my prompt' already exists`, and will return `A prompt named ` instead.